pkg/deploy/assets/rp-production.json

```diff
@@ -539,6 +539,9 @@
             "sku": {
                 "name": "Standard_LRS"
             },
+            "properties": {
+                "allowSharedKeyAccess": false
+            },
             "location": "[resourceGroup().location]",
             "name": "[substring(parameters('storageAccountDomain'), 0, indexOf(parameters('storageAccountDomain'), '.'))]",
             "type": "Microsoft.Storage/storageAccounts",
```

Comment:
should we update the apiVersion in L548 as well?

Reply:
Yes! Now that I took a better look at `apiVersions`, I understood that it must be updated.